src/node/install.sh

```diff
@@ -370,6 +378,32 @@ if [ ! -z "${ADDITIONAL_VERSIONS}" ]; then
 fi
 
 # Install pnpm
+# Declare an associative array
+declare -A node_pnpm_map
+
+# Populate the associative array with Node.js version to pnpm version mappings
+node_pnpm_map["10.13.0"]="5.18.10"
```

Comment:
Thanks for adding it. I was hoping if there was a way to to dynamically find the compatible version without hardcoding, however, seems like there isn't a way.

Can we revert these changes? Let's add a new "pnpmVersion" Feature option where users can pin the `pnpm` version if they need to fix the compatibility issues.

Hence, in the test case which fails, we would use `pnpmVersion` to fix it.